docs/Concepts/Privacy/PrivateTransactionManager.md

```diff
@@ -6,14 +6,47 @@ private transactions.
 The Private Transaction Manager is a separate component that is concerned with the storing and distribution
 of encrypted private transaction data between recipients of a private transaction.
 
+The Private Transaction Manager has two distinct components:
+
+* Transaction Manager
+* Enclave
+
 To enable private transactions, use the `PRIVATE_CONFIG` environment variable when starting a GoQuorum
-node to provide the node with the path to the Privacy Manager's `.ipc` socket, e.g.:
+node to provide the path to the Privacy Manager's `.ipc` socket.
+
+Use the `PRIVATE_CONFIG` environment variable either with a direct path to the IPC socket or with a path to a configuration file as explained in the following sections. 
+
+## Direct IPC connection configuration
+
+This is the basic method for specifying the IPC socket:
 
 ```bash
 export PRIVATE_CONFIG=path/to/tm.ipc
 ```
 
-The Private Transaction Manager has two components:
+## Using an IPC connection configuration file
 
-* Transaction Manager
-* Enclave
+Use a TOML configuration file can to specify more options of the IPC socket:
```

Comment:
Mistake in the grammar - I think it should read:
"Use a TOML configuration file to specify more options for the IPC socket:"

Reply:
Yes will fix that, I'm making changes right now, ignore for the next 20 mins ;)